test/functional/specs/actions/applyPropositions.spec.js

```diff
@@ -240,7 +240,6 @@ test("doesn't show error for viewName value that is not a data element", async (
     extensionSettings: mockExtensionSettings
   });
   await viewNameField.typeText("%dataelement");
-  await extensionViewController.expectIsValid();
```

Comment:
This view is not valid when it is empty. But the test just tests just a sepecific field that it doesn't show an error.